apps/okreads-e2e/src/specs/reading-list.spec.ts

```diff
@@ -17,4 +17,62 @@ describe('When: I use the reading list feature', () => {
       )
     );
   });
+
+  it('Then: I should see my reading list and snackbar after adding book to Reading List', async () => {
+    const numberOne = 1;
+    await browser.get('/');
+    await browser.wait(
+      ExpectedConditions.textToBePresentInElement($('tmo-root'), 'okreads')
+    );
+    const form = await $('form');
+    const input = await $('input[type="search"]');
+    await input.sendKeys('NgRx');
+    await form.submit();
+    const readingListBooks = await $$('[data-testing="book-item"]');
+    expect(readingListBooks.length).toBeGreaterThan(numberOne);
```

Comment:
This is a nitpick, but I don't think a constant was needed for this; I would just say `.toBeGreaterThan(1);`. Simplicity is always better :)